docs/reference/create-store.md

```diff
@@ -0,0 +1,134 @@
+---
+title: createStore
+description:
+nav: 202
+---
+
+# createStore
+
+`createStore` lets you create a vanilla store with store API utilities. In React, you can utilize a
+vanilla store using the [`useStore`](./use-store) hook.
+
+```js
+createStore(initializer)
+```
+
+- [Reference](#reference)
+  - [Signature](#createstore-signature)
+  - [`setState` function](#setstate-function)
+  - [`getState` function](#getstate-function)
+  - [`subscribe` function](#subscribe-function)
+  - [`storeApi`](#storeapi)
+- [Usage](#usage)
+  - [Updating state based on previous state](#updating-state-base-on-a-previous-state)
+  - [Updating objects and non-objects in state](#updating-objects-and-non-objects-in-state)
+  - [Subscribing to state updates](#subscribing-to-state-updates)
+  - [Split state in multiple slices](#split-state-in-multiple-slices)
+- [Troubleshooting](#troubleshooting)
+  - [I’ve updated the state, but the screen doesn’t update](#ive-updated-the-state-but-the-screen-doesnt-update)
+
+## Reference
+
+### `createStore` Signature
+
+```ts
+createStore<T>()(initializer: StateCreator<T, [], []>): StoreApi<T>
+```
+
+#### Parameters
+
+- `initializer`: The value you want the state to be initially. It can be a value of any type, but
+  when you pass a function should take `setState` function, `getState` function and `storeApi` as
+  arguments.
+
+#### Returns
+
+`createStore` returns a vanilla store with some API utilities. These API utilities are: `setState`
+function, `getState` function, and `subscribe` function.
+
+### `setState` function
```

Comment:
I feel like such functions like `setState` should be in one place, with hyperlinks to them from, e.g., `combine` function to avoid copying.

Reply:
@sewera that's not supported with the current stuff but when we migrate to vitepress we can move everything in one place and then just import them